js/adapt-contrib-mcq.js

```diff
@@ -213,7 +213,12 @@ define(function(require) {
 
             // Check if correct answers matches correct items and there are no incorrect selections
             var answeredCorrectly = (numberOfCorrectAnswers === numberOfRequiredAnswers) && (numberOfIncorrectAnswers === 0);
-            return answeredCorrectly;
+            if(answeredCorrectly) {
+                return answeredCorrectly;
+            } else {
+               //Check if _requiredCorrectAnswers is less than or matches numberOfCorrectAnswers and there are no incorrect selections
+               return (this.model.get('_requiredCorrectAnswers') <= numberOfCorrectAnswers) && (numberOfIncorrectAnswers === 0);
```

Comment:
@andrewhosgood makes a good point, this changes the default behaviour of the mcq.
can we:
```javascript
if (answeredCorrectly) {
   // Correct
   return true;
}
if (this.model.get('_requiredCorrectAnswers') == -1) {
   //Incorrect with no specified _requiredCorrectAnswers, assumes _selectable 
   return false;
}
//Check if _requiredCorrectAnswers is less than or matches numberOfCorrectAnswers and there are no incorrect selections
return (this.model.get('_requiredCorrectAnswers') <= numberOfCorrectAnswers) && (numberOfIncorrectAnswers === 0);
```

with default = -1 ?